Add Command Suit Cycler

Ryan180602 opened the pull request:
:cl: Ryan180602
rscadd: Add Command Suit Cycler to Bridge EVA
/:cl:

SierraKomodo reviewed (dismissed):
Compilation errors caused by using map-specific vars in global code spaces - Move the cycles for map-specific stuff to `maps/torch` and include them in the torch mapfile instead of the main .dme to fix this.

```
 code/game/machinery/suit_cycler_units.dm:81:error: access_explorer: undefined var
code/game/machinery/suit_storage_units.dm:122:error: /obj/item/clothing/suit/space/void/exploration: undefined type path
code/game/machinery/suit_storage_units.dm:123:error: /obj/item/clothing/head/helmet/space/void/exploration: undefined type path
code/game/machinery/suit_storage_units.dm:127:error: access_explorer: undefined var
code/game/machinery/suit_storage_units.dm:137:error: access_pilot: undefined var
code/game/machinery/suit_storage_units.dm:142:error: /obj/item/clothing/suit/space/void/command: undefined type path
code/game/machinery/suit_storage_units.dm:143:error: /obj/item/clothing/head/helmet/space/void/command: undefined type path
```

SierraKomodo commented:
> 
> 
> Compilation errors caused by using map-specific vars in global code spaces - Move the cycles for map-specific stuff to `maps/torch` and include them in the torch mapfile instead of the main .dme to fix this.
> 
> ```
>  code/game/machinery/suit_cycler_units.dm:81:error: access_explorer: undefined var
> code/game/machinery/suit_storage_units.dm:122:error: /obj/item/clothing/suit/space/void/exploration: undefined type path
> code/game/machinery/suit_storage_units.dm:123:error: /obj/item/clothing/head/helmet/space/void/exploration: undefined type path
> code/game/machinery/suit_storage_units.dm:127:error: access_explorer: undefined var
> code/game/machinery/suit_storage_units.dm:137:error: access_pilot: undefined var
> code/game/machinery/suit_storage_units.dm:142:error: /obj/item/clothing/suit/space/void/command: undefined type path
> code/game/machinery/suit_storage_units.dm:143:error: /obj/item/clothing/head/helmet/space/void/command: undefined type path
> ```

Actually looking at the diff - These were originally already in Torch map-specific files, so just put them back where they were and that'll be fine.

Ryan180602 committed:
cmdcycler

also moves the rest of suit related machinery to their respective dms because why the fuck would there be half here half there
a

Ryan180602 commented:
reverted that refactor
